data-science-pipelines-operator/base/params.env

```diff
@@ -0,0 +1,8 @@
+IMAGES_APISERVER=registry.redhat.io/rhods/odh-ml-pipelines-api-server-rhel8:latest
+IMAGES_ARTIFACT=registry.redhat.io/rhods/odh-ml-pipelines-artifact-manager-rhel8:latest
+IMAGES_PERSISTENTAGENT=registry.redhat.io/rhods/odh-ml-pipelines-persistenceagent-rhel8:latest
+IMAGES_SCHEDULEDWORKFLOW=registry.redhat.io/rhods/odh-ml-pipelines-scheduledworkflow-rhel8:latest
+IMAGES_VIEWERCRD=registry.redhat.io/rhods/odh-ml-pipelines-viewercontroller-rhel8:latest
+IMAGES_CACHE=registry.redhat.io/ubi8-minimal:8.7
```

Comment:
think it is something like "registry.redhat.io/rhods/odh-ml-pipelines-cache-rhel8"

Reply:
I think so too, but I'll like to confirm - will circle back

Reply:
updated